ouroboros-consensus/src/Ouroboros/Consensus/Node.hs

```diff
@@ -203,12 +212,18 @@ data LowLevelRunNodeArgs m addrNTN addrNTC versionDataNTN versionDataNTC blk = L
     , llrnMaxClockSkew :: ClockSkew
     }
 
+-- | P2P Switch
+--
+data NetworkP2PMode = EnabledP2PMode
+                    | DisabledP2PMode
+  deriving (Eq, Show)
```

Comment:
```suggestion
data NetworkMode = P2PMode
                 | FederatedMode
  derivng (Eq, Ord)
```
Maybe let's use on slack for other ideas.

Reply:
Maybe `Federated` and `P2P` are also nicer suffixes than `NonP2P` and `P2P` (but it would be a pervasive change).